public/static-checkout/css/style.css

```diff
@@ -210,11 +192,6 @@ header {
   transform: translate(0%, 25%);
 }
 
-.grid-container {
-  width: 100%;
-  max-width: 1380px;
```

Comment:
How does it look on larger than laptop screen size?

Reply:
Also the .row class keeps the container inside the padding (bottom now has no padding). You're correct that the :before :after is not used right now, but we can keep the .row or add margin-bottom to the .container-fluid and remove the <div class="row"> on the HTML